package NKikimrArrowSchema;

message TSchemaSubset {

    message TFieldsList {
        repeated uint32 FieldsIdx = 1;
    }

    oneof Implementation {
        TFieldsList List = 1;
        std::string Bits = 2;

    }
}